Model/Config.php

```diff
@@ -134,7 +134,7 @@ public function updateBalancePayStatus($scope = ScopeInterface::SCOPE_STORE, $st
      * @param string $scope
      * @param int $storeId
      */
-    public function updateCustomerGroup($scope = ScopeInterface::SCOPE_STORE, $value, $storeId = null)
+    public function updateCustomerGroup($scope = ScopeInterface::SCOPE_STORE, $value, int $storeId = 0)
```

Comment:
Why do you have default value here for store id? what is the use case when we update customer group with store id = 0?

Reply:
It will update for all stores, which is magento default. To save for global configuration.